ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiPartyManagementService.scala

```diff
@@ -57,6 +59,7 @@ private[apiserver] final class ApiPartyManagementService private (
   override def getParticipantId(
       request: GetParticipantIdRequest
   ): Future[GetParticipantIdResponse] = {
+    logger.info("Getting Participant ID")
```

Comment:
Is there some submission identifier that should be added to the logging context here?

Reply:
The same as above - no identifier available. What to do in such case?

Reply:
Nothing, really. I thought that was the case, but felt like commenting just to made sure. Thanks for confirming!

(same applies to other comments)

Reply:
Got it, thanks!